test/visual/rtl.html

```diff
@@ -1,28 +1,41 @@
 <!DOCTYPE html>
+<html>
+  <head lang="en">
+    <meta charset="UTF-8">
+    <title>Progress bar rtl tests</title>
+    <script>
+      document.documentElement.setAttribute('dir', 'rtl');
 
-<head lang="en">
-  <meta charset="UTF-8">
-  <title></title>
-  <script src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
-  <script>
-    const theme = window.location.search.replace(/.*theme=(\w+).*/, '$1') || 'lumo';
-    document.write(`<link rel="import" href="../../theme/${theme}/vaadin-progress-bar.html">`);
-    document.documentElement.setAttribute('dir', 'rtl');
-
-    window.addEventListener('WebComponentsReady', function() {
-      customElements.whenDefined('vaadin-progress-bar').then(function() {
-        window.ShadyDOM && window.ShadyDOM.flush(); // Force DOM composition
-        window.webComponentsAreReady = true; // Checked in gemini before capture callback
+      window.addEventListener('WebComponentsReady', function() {
+        customElements.whenDefined('vaadin-progress-bar').then(function() {
+          window.ShadyDOM && window.ShadyDOM.flush(); // Force DOM composition
+          window.webComponentsAreReady = true; // Checked in gemini before capture callback
+        });
       });
```

Comment:
Please remove this block, it does nothing. 
The `WebComponentsReady` event is dispatched by polyfill and now we removed it.